task/sast-unicode-check/0.1/sast-unicode-check.yaml

```diff
@@ -0,0 +1,274 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-unicode-check
+spec:
+  description: >-
+    Scans source code for non-printable unicode characters in all text files.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - name: image-url
+      type: string
+      description: Image URL.
+      default: ""
+    - name: image-digest
+      type: string
+      description: Image digest to report findings for.
+      default: ""
+    - name: FIND_UNICODE_CONTROL_GIT_URL
+      type: string
+      description: URL from repository to find unicode control.
+      default: "https://github.com/siddhesh/find-unicode-control.git#c2accbfbba7553a8bc1ebd97089ae08ad8347e58"
+    - name: FIND_UNICODE_CONTROL_ARGS
+      type: string
+      description: arguments for find-unicode-control command.
+      default: "-p bidi -v -d -t"
+    - name: KFP_GIT_URL
+      type: string
+      description: URL from repository to download known false positives files.
+      # FIXME: Red Hat internal projects will default to https://gitlab.cee.redhat.com/osh/known-false-positives.git when KONFLUX-4530 is resolved
+      default: ""
+    - name: PROJECT_NVR
+      type: string
+      default: ""
+      description: |
+        Name-Version-Release (NVR) of the scanned project.
+        It is used to find path exclusions (it is optional).
+    - name: RECORD_EXCLUDED
+      type: string
+      default: "false"
+      description: |
+        Whether to record the excluded findings (defaults to false).
+        If `true`, the the excluded findings will be stored in `excluded-findings.json`.
+  volumes:
+    - name: trusted-ca
+      configMap:
+        name: $(params.caTrustConfigMapName)
```

Comment:
I think these params missing from the `params` section and should be added

Reply:
@jperezdealgaba Do we need any secrets for the `sast-unicode-check` actually?  Are they needed to configure the CA certificates?

Reply:
Hey! I am not sure if I understand your question. 
They are not being added as secret but as a `configmap`. In the task provided by Konflux team, they are adding it as such: 
https://github.com/konflux-ci/build-definitions/blob/dc50185eed271866a7fc4f80aaa8285fd86423af/task/prefetch-dependencies/0.1/prefetch-dependencies.yaml#L142-L147 . 

This configmap is later mounted as a volume in order to be able to use it and install the certificates.
Secrets can be used for the same purpose, but I wouldn't deviate from the Konflux implementation. 
Does that answer your question?

Reply:
@jperezdealgaba You said:
> The volumeMounts section is also missing to mount the secrets

... which made me wondering which **secrets** are we talking about in the context of `sast-unicode-check`.

As I understand it, no secrets should be needed to run this task.

Reply:
Right. Sorry for the typo. `secrets` = `configmap`.
Edited initial comment

Reply:
Got it, thank you for clarifying it!

Reply:
@jperezdealgaba yes, you are right, I have added params: `caTrustConfigMapName` and `caTrustConfigMapKey`